rainmaker_orchestrator/server.py

```
@@ -1,426 +1,714 @@
     """
     try:
-        workspace = orchestrator.fs.workspace
-        if not os.path.exists(workspace):
-            return jsonify({
-                "files": [],
-                "workspace": workspace
-            }), 200
+        workspace_path = Path(settings.workspace_path) / path
+
+        if not workspace_path.exists():
+            raise HTTPException(status_code=404, detail="Path not found")
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)